src/app/views/sidebar/resource-explorer/collection/CollectionPermissions.tsx

```diff
@@ -43,6 +43,13 @@ const CollectionPermissions: FC<PopupsComponent<null>> = (props) => {
     trackDownload(filename, componentNames.DOWNLOAD_COLLECTION_PERMISSIONS_BUTTON);
   }
 
+  function handleTelemetryClick(e: React.MouseEvent<HTMLElement | HTMLAnchorElement | HTMLButtonElement, MouseEvent>) {
+    telemetry.trackLinkClickEvent(
+      (e.currentTarget as HTMLAnchorElement).href,
+      componentNames.MICROSOFT_GRAPH_PERMISSIONS_REFERENCE_DOCS_LINK
+    );
+  }
+
```

Comment:
Let's also make this an arrow func 
`const handleTelemetryClick = (e: React.MouseEvent<HTMLElement | HTMLAnchorElement | HTMLButtonElement, MouseEvent>) => {
  telemetry.trackLinkClickEvent(
    (e.currentTarget as HTMLAnchorElement).href,
    componentNames.MICROSOFT_GRAPH_PERMISSIONS_REFERENCE_DOCS_LINK
  );
};`